### PR TITLE
[AKS] `az aks operation-abort` and `aks nodepool operation-abort`: Add aks, and nodepool `operation-abort` commands

### PR DESCRIPTION
**Pull request by charliedmcb**
**Related command**

`aks`

**Description**

GA'ing the abort feature for AKS on managed cluster, and agentpool.

This PR is simply porting over the following PR from preview:
https://github.com/Azure/azure-cli-extensions/pull/5271

The following changes were made in the conversion from preview to GA:
1. I updated the get_models logic (seems in alignment) with the GA repo
2. removed --aks-custom-headers from both the nodepool, and mc abort operations, as its unneeded for the command, and trimming aware the CLI to be as sleek as possible for GA makes sense to me, even if its breaking between preview and GA
3. abort_latest_operation didn't exist on the client, so switched to begin_abort_latest_operation
4. removed `--nodepool-name` param from the nodepool abort, since its already handled by the parent subcommand scope.

**Testing Guide**

Abort for managed cluster:
`aks operation-abort --resource-group={resource_group} --name={name}`

Abort for agentpool:
`aks nodepool operation-abort --resource-group={resource_group} --cluster-name={name} --nodepool-name={node_pool_name}`

**History Notes**

[AKS] `aks nodepool operation-abort`: Add new command to support aborting last running operation on nodepool
[AKS] `aks operation-abort`: Add new command to support aborting last running operation on managed cluster

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
AKS

**Comment by FumingZhang** [comment]
Queued a [pipeline](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=70050846&view=results) to validate the newly added test cases.

File: src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py [hunk 1]
```diff
@@ -4884,6 +4885,68 @@ def test_aks_stop_and_start(self, resource_group, resource_group_location):
...
+        time.sleep(10)
+        show_cmd = 'aks show --resource-group={resource_group} --name={name}'
+        self.cmd(show_cmd, checks=[self.check('provisioningState', 'Canceled')])
+
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
+    def test_aks_nodepool_abort(self, resource_group, resource_group_location):
+        aks_name = self.create_random_name('cliakstest', 16)
+        node_pool_name = self.create_random_name('c', 6)
+        node_vm_size = 'standard_d4darm_v3'
```
Comment (FumingZhang) [inline src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py:4913]: Could you please choose a common vm size (or not set to use the default vm size Standard_DS2_v2)? Sadly, there's no quota for this vm size in the cli test subs.

Comment (charliedmcb) [reply src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py:4913]: Updated to explicitly `Standard_DS2_v2`

File: src/azure-cli/azure/cli/command_modules/acs/custom.py [hunk 2]
```diff
@@ -2248,6 +2248,51 @@ def aks_agentpool_delete(cmd, client, resource_group_name, cluster_name,
     return sdk_no_wait(no_wait, client.begin_delete, resource_group_name, cluster_name, nodepool_name)
 
 
+def aks_agentpool_operation_abort(cmd,   # pylint: disable=unused-argument
```
[the remark this answers is not in the document]
Comment (charliedmcb) [reply src/azure-cli/azure/cli/command_modules/acs/custom.py:2251]: Removed

File: src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py (hunk 1 above)
Comment (FumingZhang) [reply src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py]: Queued another [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=70250915&view=results).

**Review by FumingZhang — APPROVED** [review]
LGTM

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**